### ISSUE TITLE
[Suggestion] Option to ignore directory.

### ISSUE THREAD

**Issue by Xotchkass** [DanielGavin/ols#552]
So because the standard `base' package uses reserved package names (obviously), but ols still checks it and reports it as an error, the moment I jump there to look up a signature or search for something, it fills my editor with red. 

![изображение](https://github.com/user-attachments/assets/d6a9403b-4625-413e-a6bc-c4532cdfea20)

Not the end of the world, but quite annoying. Maybe you can add a setting like `ignore path` or something?

**Comment by DanielGavin** [comment]
Yeah all the reserved packages can't actually be parsed by the parser right now because of it. It should really just never try to parse them.

**Comment by DanielGavin** [comment]
Should be fixed.
